package.json

```diff
@@ -95,7 +95,7 @@
     "@trysound/sax": "0.1.1",
     "chalk": "^4.1.0",
     "commander": "^7.1.0",
-    "css-select": "^3.1.2",
+    "css-select": "^4.1.2",
```

Comment:
Shouldn't it be 4.1.3 considering the conversation in here https://github.com/fb55/css-select/issues/468

Reply:
4.1.3 was released a day after this PR was opened :)

Reply:
Oh, 🙂 well, it's still not too late to change it

Reply:
Thanks for pointing this out @seahindeniz, I updated the Pull Request :+1: